src/hooks/useQuery.ts

```diff
@@ -68,16 +60,18 @@ export const useQuery = <T = any, V = object>(
         })
       );
     },
-    [args.pause, args.requestPolicy, client, request, setState]
+    [args.requestPolicy, client, request, setState]
   );
 
-  // This calls executeQuery immediately during the initial mount and
```

Comment:
Seems good but it'd be great to readd the comments and slightly rewrite them. Especially the one here that explains with this is using useImmediateEffect 😊

Reply:
Yup agreed - I'll add something back in 👍